### PR TITLE
Update: Delete old system of remove local storage

### PR DESCRIPTION
**Pull request by udelunar**

### CODE REVIEW

File: OCM/Data/Persistance/Webview/LocalStorageWebview.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//  OCM
+//
+//  Created by Eduardo Parada on 19/2/18.
+//  Copyright © 2018 Gigigo SL. All rights reserved.
+//
+
+import Foundation
+import WebKit
+
+class LocalStorageWebview {
```
Comment (jerilyngigigo) [inline OCM/Data/Persistance/Webview/LocalStorageWebview.swift]: can you rename this class to: WebviewLocalStorage plzzzz 🐱

**Comment by jerilyngigigo** [comment]
LGTM!

File: OCM/Data/Persistance/Webview/LocalStorageWebview.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            
+            guard let webSite = websiteDataTypes as? Set<String> else {
+                return
+            }
+            
+            WKWebsiteDataStore.default().removeData(ofTypes: webSite, modifiedSince: date as Date, completionHandler: {
+                
+            })
+        } else {
+            var libraryPath = NSSearchPathForDirectoriesInDomains(FileManager.SearchPathDirectory.libraryDirectory, FileManager.SearchPathDomainMask.localDomainMask, false).first!
```
Comment (jcarlosestela) [inline OCM/Data/Persistance/Webview/LocalStorageWebview.swift]: Can you remove this force-unwrapped property?

File: OCM/OCM.swift [hunk 3]
```diff
@@ -636,10 +636,10 @@ open class OCM: NSObject {
     /**
      Use it to reset the localStorage of WebView
      
-     - Since: 2.0.14
+     - Since: 2.2.2
      */
-    public func isResetLocalStorageWebView(reset: Bool) {
-        Config.resetLocalStorageWebView = reset
+    public func resetLocalStorageWebView() {
```
Comment (jcarlosestela) [inline OCM/OCM.swift]: I think that this method should be called: resetWebViewsLocalStorage(), what do u think @jerilyngigigo ?

Comment (jerilyngigigo) [reply OCM/OCM.swift]: it's a good suggestion, but maybe like this: resetWebViewLocalStorage?

**Pull request merged**